test/blocks/cai/mocks/CAIManifest.json

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
I think you have way too much unnecessary information in there. The tests were passing for me with a minimal file like this:
```json
{
  "manifestStore": {
    "activeManifest": {
      "title": "3-cr.png",
      "format": "image/png",
      "claimGenerator": "Adobe Photoshop/25.12.0 adobe_c2pa/0.12.2 c2pa-rs/0.32.5",
      "signatureInfo": {
        "issuer": "Adobe Inc.",
        "cert_serial_number": "123",
        "time": "2024-09-30T03:09:40+00:00"
      }
    }
  }
}
```
Probably still contains unused information, but I didn't bother trying to trim more.